### ISSUE TITLE
warning  'Unnecessary type declaration' from `openapi/warnings`

### ISSUE THREAD

**Issue by m-ilchuk** [bee-travels/openapi-comment-parser#33]
I have warning message  'Unnecessary type declaration' when I use: 
`@paramComponent {ExampleParameter}` and @bodyComponent

What does it mean and how to fix it ?

**Comment by bourdakos1** [comment]
I don’t think this warning is coming from openapi-comment-parser. This sounds like a typescript or eslint warning

**Comment by m-ilchuk** [comment]
I guess that's from openapi plugin, in my eslint config I have `"plugin:openapi/recommended"`. 
Error attached:
![image](https://user-images.githubusercontent.com/4456550/120227682-4d01c300-c252-11eb-987b-b9e69df38a57.png)

**Comment by bourdakos1** [comment]
Oh yea looks like a bug with the eslint plugin. Does it generate the correct openapi spec?

**Comment by m-ilchuk** [comment]
Yes, spec is correct. (only warning shows up)
